### ISSUE TITLE
confusion

### ISSUE THREAD

**Issue by yasmensarhan27** [ubsuny/23-Homework7G1#36]
what does it mean to "Label them with their energies"??

**Comment by uarif** [comment]
I think it means to title them with the respective energies. If I am not misunderstanding it. @yasmensarhan27

**Comment by WildJimmy** [comment]
Yes, you can add a label to each of the plots showing the ground state energy is my understanding

**Comment by pratibha77118** [comment]
Is that the value of final optimized potential energy, we label in our plot?
